packages/evolution-frontend/src/components/survey/__tests__/GroupWidgets.test.tsx

```
@@ -20,6 +20,21 @@ import { SurveyContext } from '../../../contexts/SurveyContext';
 jest.mock('react-markdown', () => 'Markdown');
 jest.mock('remark-gfm', () => 'remark-gfm');
 
+// Mock the ConfirmModal component, as it uses many react hooks and libraries that make jest fail
+jest.mock('chaire-lib-frontend/lib/components/modal/ConfirmModal', () => {
+    return function MockConfirmModal(props: any) {
+        if (!props.isOpen) return null;
+        return (
+            <div role="dialog" aria-label={props.title}>
+                <div>{props.title}</div>
+                <div>{props.text}</div>
+                <button onClick={props.cancelAction}>{props.cancelButtonLabel || 'Cancel'}</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)